_data/simple-icons.json

```
@@ -13665,6 +13665,11 @@
 		"hex": "801010",
 		"source": "https://github.com/mcneel/compute.rhino3d/blob/2204d998ff0397a1c6a18dd2312a96508ad48bdb/README.md"
 	},
+	{
+		"title": "Rhino Linux",
+		"hex": "8D7BE4",
+		"source": "https://raw.githubusercontent.com/rhino-linux/unicons/1d9a60e9ad65878b89074f0ed06215c6ed48db85/scalable/emblems/rhinolinux-symbolic.svg"
```

```suggestion
		"source": "https://github.com/rhino-linux/unicons/blob/1d9a60e9ad65878b89074f0ed06215c6ed48db85/scalable/emblems/rhinolinux-symbolic.svg"
```

Let's use permalink instead.